.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js: "stable"
+branches:
+  only:
+    - develop
```

Comment:
Hey @amilykassim why specify only `develop` to be built?

Reply:
> Hey @amilykassim why specify only `develop` to be built?

@Paulstar200 I specified only develop to be built because it was the main branch but if it is not a best practice I can remove it.

Reply:
@amilykassim set it to build all our branches.

Reply:
> @amilykassim set it to build all our branches.

OK, let me do that @Paulstar200